tests/test_delete_all_links.py

```diff
@@ -1,16 +1,25 @@
 import requests
+import allure
```

Comment:
Imports should sorted by alphabet.

Reply:
Fixed in 034a15a5623401f2633bbd56277fe0c11ba2327c